app/javascript/chat/chat.jsx

```diff
@@ -599,6 +606,18 @@ export default class Chat extends Component {
         expanded: window.innerWidth > 600,
       });
     }
+    if (upArrowPressed && messageIsEmpty) {
```

Comment:
Up arrow is used to editing previous message in Slack: I think having it lead to a destructive act here will be jarring for users. Perhaps listen for the 'delete' key?

Reply:
I made the change to use the up arrow as a shortcut to edit the last message and as suggested added the delete key as a shortcut to delete the last message.